example/Index.re

```diff
@@ -1,11 +1,24 @@
 open Express;
 
-let __dirname: option string = [%bs.node __dirname];
+let app = express ();
 
-let app = Express.express ();
+App.useOnPath app path::"/" @@ Middleware.fromF (fun _ _ next => {
+  Js.log "Request received"; 
+     /* This will be printed for every request */
+  next Js.undefined [@bs]
```

Comment:
And shouldn't there be a nicely named function for `next Js.undefined` too, like `next router` and `next error e`?